tokens/rounded.json

```
@@ -0,0 +1,24 @@
+{
+  "rounded": {
+    "none": {
+      "value": "0",
+      "themeable": true
+    },
+    "smallest": {
+      "value": "2px",
```

Please also move the descriptions. Even if this leads to very long lines, i'd suggest to just put everything on one line like so:

```suggestion
      "$description": "Smallest available radius discernible from 0. Use this value for rounded elements inside elements that have {rounded.small} applied as border-radius.",
      "value": "2px",
```

---

Done in https://github.com/demos-europe/demosplan-ui/pull/193/commits/f3a4ac024df652350d44b5744a15e701a3378953